pyzoo/zoo/friesian/feature/table.py

```
@@ -91,6 +97,21 @@ def broadcast(self):
         """
         self.df = broadcast(self.df)
 
+    def select(self, *cols):
+        """
+        Select specific columns
+
+        :param cols: a string or a list of strings that specifies column names. If it is '*',
+        select all the columns. If it is none, raise error
+
+        :return: A new Table that contains the specified columns.
+        """
+        # spark will return an empty table when cols is None,
+        # therefore we raise the error if cols is None
```

Change to `If cols is None, it makes more sense to raise error instead of returning an empty Table.`